source/slang/slang-session.cpp

```
@@ -59,6 +59,16 @@ Linkage::Linkage(Session* session, ASTBuilder* astBuilder, Linkage* builtinLinka
     , m_cmdLineContext(new CommandLineContext())
     , m_stringSlicePool(StringSlicePool::Style::Default)
 {
+    // [TEST] Deliberate heap-buffer-overflow to verify sanitizer CI catches it.
+    // Remove this block after verification.
+    {
```

Cppcheck (2.20.0)

[information] Include file

(missingInclude)